### PR TITLE
NSQ read/write/relay support

### PR DESCRIPTION
**Pull request by blinktag**

### CODE REVIEW

File: backends/nsq/read.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
+package nsq
+
+import (
+	"sync"
+
+	"github.com/batchcorp/plumber/reader"
```
Comment (dselans) [inline backends/nsq/read.go:6]: organize imports

File: cli/nsq.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+	cmd.Flag("insecure-tls", "Whether to verify server certificate").
+		Envar("PLUMBER_RELAY_NSQ_SKIP_VERIFY_TLS").
+		Default("false").
+		BoolVar(&opts.NSQ.InsecureTLS)
+}
+
+// addReadNSQFlags creates flags used for reading from NSQ
+func addReadNSQFlags(cmd *kingpin.CmdClause, opts *Options) {
+	cmd.Flag("channel", "Channel").
```
Comment (dselans) [inline cli/nsq.go:87]: Can you read from more than 1 channel with the lib? If yes, we should do the same for relay as well.

**Review by dselans — APPROVED** [review]
README.md updates + ENV var docs update for new relay flags.

Noted somewhere as well - if the main client lib supports reading from multiple channels (topics?) - we should add that.

File: backends/nsq/read.go (hunk 1 above)
Comment (blinktag) [reply backends/nsq/read.go:6]: Fixed!

File: cli/nsq.go (hunk 2 above)
Comment (blinktag) [reply cli/nsq.go:87]: Lib only supports single channel/topic per consumer

**Pull request merged**